### ISSUE TITLE
About the WDMC loss function

### ISSUE THREAD

**Issue by silence-tang** [lehaifeng/DASNet#49]
I have two questions about the ContrastiveLoss1() loss function in _loss.py_
1. I think
`mdist_pos = torch.clamp(dist-self.margin1, min=0.0)`
`mdist_neg = torch.clamp(self.margin2-dist, min=0.0)`
should be 
`mdist_pos = torch.clamp(self.margin1-dist, min=0.0)`
`mdist_neg = torch.clamp(dist-self.margin2, min=0.0)`
2. I think the original code forgot to add the weights mentioned in the paper, so it seems like it should be
`w1 = 1 / 0.147
w2 = 1 / (1 - 0.147)
loss_pos = w2 * ((1 - y) * (mdist_pos.pow(2)))  # w2小，使网络较不关注unchanged pairs
loss_neg = w1 * (y * (mdist_neg.pow(2)))         # w1大，使网络重点关注changed pairs`
